pkg/api/vlabs/types.go

```diff
@@ -202,6 +203,9 @@ type KeyVaultCertificate struct {
 // OrchestratorType defines orchestrators supported by ACS
 type OrchestratorType string
 
+// OrchestratorVersion defines the version for orchestratorType
+type OrchestratorVersion string
```

Comment:
to detect serialization / deserialization issues, please add some version tests with a variety of versions to pkg/acsengine/testdata/